### PR TITLE
Change home background

### PR DESCRIPTION
**Pull request by GuidoCaldara**
Hey @rayhanw  I changed the background color .....
image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -11,3 +11,7 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+body {
+  background-color: #ff8f8f;
```
Comment (rayhanw) [inline app/assets/stylesheets/application.scss:16]: I think you should change the color to rainbow instead

Comment (GuidoCaldara) [reply app/assets/stylesheets/application.scss:16]: how can I do rainbow??!?!

**Comment by rayhanw** [comment]
gonna merge it, loosk good

**Pull request merged**